### PR TITLE
Update js reader dependency

### PR DESCRIPTION
**Pull request by digitalsadhu**
## JIRA Issue
[COREWEB-120](https://jira.finn.no/browse/COREWEB-120)

## Status
**READY**

## Description
Updates js reader. This changes bundles structurally though meaning remains the same.

## Todos
- [x] Tests

### CODE REVIEW

File: test/utils.test.js [hunk 1]
```diff
@@ -4,9 +4,6 @@ beforeEach(() => jest.resetModules());
 
 test('bundleFeeds() throws error when bundling JS', async () => {
     expect.hasAssertions();
-    jest.doMock('@asset-pipe/js-reader', () => () =>
-        Promise.reject(new Error())
-    );
     const { bundleFeeds, endWorkers } = require('../lib/utils');
```
Comment (SimenB) [inline test/utils.test.js]: does this `require` still have to be inline?

Comment (digitalsadhu) [reply test/utils.test.js]: unsure, will test out.

Comment (digitalsadhu) [reply test/utils.test.js]: most likely yes since i needed to keep `beforeEach(() => jest.resetModules());`

Comment (SimenB) [reply test/utils.test.js]: why? if we don't mock anymore

Comment (digitalsadhu) [reply test/utils.test.js]: yup, seems so

Comment (digitalsadhu) [reply test/utils.test.js]: because new Worker is called at the to of the utils module and multiple calls to endWorker break things

Comment (digitalsadhu) [reply test/utils.test.js]: i think it probably should have all been inside a class or closure

**Pull request merged**